packages/foundry/contracts/utils/ContangoWand.sol

```diff
@@ -0,0 +1,208 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity >=0.8.13;
+
+import "../interfaces/ICauldronGov.sol";
+import "../interfaces/ICauldron.sol";
+import "../interfaces/ILadleGov.sol";
+import "../interfaces/ILadle.sol";
+import "../interfaces/IJoin.sol";
+import "../oracles/yieldspace/YieldSpaceMultiOracle.sol";
+import "../oracles/composite/CompositeMultiOracle.sol";
+import "@yield-protocol/yieldspace-tv/src/interfaces/IPool.sol";
+import "@yield-protocol/utils-v2/src/access/AccessControl.sol";
+
+/// @title A contract that allows configuring the cauldron and ladle within bounds
+contract ContangoWand is AccessControl {
+    ICauldronGov public immutable cauldron;
```

Comment:
we should call this `contangoCauldron` so the rest of the code is easier to understand